symfony-cmf/cmf-phpcr-dbal-pack/1.0/config/packages/cmf_phpcr_dbal.yaml

```diff
@@ -0,0 +1,24 @@
+framework:
+    templating: { engines: ['twig'] }
+
+cmf_menu:
+    persistence:
+        phpcr: ~
+
+cmf_routing:
+    chain:
+        routers_by_id:
+            cmf_routing.dynamic_router: 200
+            router.default:             100
+    dynamic:
+        persistence:
+            phpcr:
+                enabled: true
+
+cmf_content:
+    persistence:
+        phpcr: ~
+
+cmf_core:
+    multilang:
+        locales: [en,de]
```

Comment:
this looks more like a demo example than a most common configuration. i think the pack should provide a minimal useful configuration, not contain demo things. as its an important feature, it could be there but commented out with some short explanation. afaik that is ok for recipies to provide commented out examples for the most important things.

Reply:
>  i think the pack should provide a minimal useful configuration

Yes

> afaik that is ok for recipies to provide commented out examples for the most important things.

Generally no. I would rather see a link to some documentation. In rare cases I make exceptions, say when there is mandatory to add some config but the exact value is impossible to guess.

Reply:
So i would eventually point this one to an parameter array which is empty here.

Reply:
So i would eventually point this one to an parameter array which is empty here.